crt_portal/cts_forms/views.py

```diff
@@ -471,6 +471,7 @@ def process_intake_filters(request):
         'date_range_start': start_date,
         'date_range_end': end_date,
         'activity_count': len(reports),
+        'response_count': len(response_actions),
```

Comment:
iiuc `count` might be better than `len` here (for both your new line and the existing one)?

https://docs.djangoproject.com/en/dev/ref/models/querysets/#when-querysets-are-evaluated